spanner/api/SpannerTest/Tests.cs

```diff
@@ -96,19 +101,14 @@ public void Dispose()
             catch (RpcException ex) when (ex.Status.StatusCode == StatusCode.NotFound) { }
         }
 
-        public string ProjectId { get; private set; } =
-            Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        public string ProjectId { get; private set; } = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
         // Allow environment variables to override the default instance and database names.
         public string InstanceId { get; private set; } =
             Environment.GetEnvironmentVariable("TEST_SPANNER_INSTANCE") ?? "my-instance";
-        private static readonly string s_randomDatabaseName = "my-db-"
-            + TestUtil.RandomName();
-        private static readonly string s_randomToBeCancelledBackupName = "my-backup-"
-            + TestUtil.RandomName();
-        private static readonly string s_randomRestoredDatabaseName = "my-db-"
-            + TestUtil.RandomName();
-        public string DatabaseId =
-            Environment.GetEnvironmentVariable("TEST_SPANNER_DATABASE") ?? s_randomDatabaseName;
+        private static readonly string s_randomDatabaseName = $"my-db-{DateTime.UtcNow.ToTimestamp().Seconds}";
+        private static readonly string s_randomToBeCancelledBackupName = $"my-backup-{DateTime.UtcNow.ToTimestamp().Seconds}";

```

Comment:
Seconds will go from 0 to 59, you might want TotalSeconds or even TotalMilliseconds, or better still, format the date to something like "yyyyMMddHHmmss" so it's quickly readable and if we need to look for something that has failed we can find it quickly.
Here and in all the others.

Reply:
seconds of UTC time since Unix epoch 1970-01-01T00:00:00Z. Must be from 0001-01-01T00:00:00Z to 9999-12-31T23:59:59Z inclusive.

Reply:
it starts from `new DateTime(1970, 1, 1,0, 0, 0, DateTimeKind.Utc)`

Reply:
Oh I see, you are calling ToTimestamp(), which I believe is a an extension method defined in Protobuf.WellKnownTypes.
So, I would prefer not to use protobuf code to do something that we can do with .NET: how about `DateTimeOffset.UtcNow.ToUnixTimeSeconds();`